packages/engine-rn/templates/platforms/android/app/src/main/java/rnv_template/MainActivity.kt

```diff
@@ -1,10 +1,16 @@
 package {{APPLICATION_ID}}
 
+import android.os.Bundle
```

Comment:
do we need these for every app?

Reply:
We define the onCreate method in MainActivity.kt and pass the Bundle. I can make this whole block dynamic. In this case, we don't need this import for all applications.